frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/utilization-card/prometheus-multi-utilization-item.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { useUtilizationDuration } from '@console/shared/src/hooks/useUtilizationDuration';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { getPrometheusQueryResponse } from '@console/internal/actions/dashboards';
+import { Humanize } from '@console/internal/components/utils';
+
+import { getRangeVectorStats } from '@console/internal/components/graphs/utils';
+import { MultilineUtilizationItem } from './multi-utilization-item';
+
+export const PrometheusMultilineUtilizationItem = withDashboardResources<
```

Comment:
I think we should not require one more component just to fetch prometheus queries. We could use `usePrometheusQuery` hook to get those values.